### PR TITLE
Update version of Polaris in "CDN Styles" example

### LINKED ISSUE #3050: CDN-styles readme is outdated and references Polaris 4.0.0 which doesn't work with examples, needs to be updated to 4.25.0

### PR DESCRIPTION
**Pull request by kaelig**
Fixes #3050

### CODE REVIEW

File: examples/cdn-styles/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The CDN hosted CSS file needs to be included in your HTML document. Replace the
 ```html
 <link
   rel="stylesheet"
-  href="https://unpkg.com/@shopify/polaris@4.0.0/styles.min.css"
+  href="https://unpkg.com/@shopify/polaris@4.26.1/styles.min.css"
```
Comment (chloerice) [inline examples/cdn-styles/README.md:14]:
```suggestion
  href="https://unpkg.com/@shopify/polaris/styles.min.css"
```

File: examples/cdn-styles/index.html [hunk 2]
```diff
@@ -1,255 +1,473 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Polaris CDN Styles</title>
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@shopify/polaris@4.26.1/styles.min.css"
```
Comment (chloerice) [inline examples/cdn-styles/index.html:10]:
```suggestion
      href="https://unpkg.com/@shopify/polaris/styles.min.css"
```

**Review by chloerice — DISMISSED** [review]
Nice catch, Kaelig! It looks like this can be kept up to date automatically if we [remove the version tag entirely](https://unpkg.com/) 🤓

File: examples/cdn-styles/index.html (hunk 2 above)
Comment (BPScott) [reply examples/cdn-styles/index.html:10]: This is gonna make the example more brittle, as we publish changes that are breaking when you're using CSS styles only in minor versions (e.g. adding a wrapping div somewhere).

Comment (kaelig) [reply examples/cdn-styles/index.html:10]: I agree, pointing to a specific version makes these examples less likely to unexpectedly break.

**Comment by kaelig** [comment]
@chloerice Ben and I had thoughts on the suggested change. How do you feel about this?

**Review by kvendrik — APPROVED** [review]
LGTM but I'm wondering if the formatting changes you made to [`index.html`](https://github.com/Shopify/polaris-react/blob/f668310d337ec277a46daf3f72245aa35f7afa90/examples/cdn-styles/index.html) were intentional? It looks like your formatter might've formatted the file on save.

**Comment by kaelig** [comment]
Thank you @kvendrik – the formatting changes were intentional, and there also were some actual markup changes.

**Comment by kaelig** [comment]
I updated the markup and version of the CDN file to v5.0.0.

**Review by asawm — COMMENTED** [review]
<script type="text/javascript">
    adroll_adv_id = "MV32I2JLVBACDNKK2I6AYV";
    adroll_pix_id = "DM6G2VHHTVDD3EXJ55FNLM";
    adroll_version = "2.0";

    (function(w, d, e, o, a) {
        w.__adroll_loaded = true;
        w.adroll = w.adroll || [];
        w.adroll.f = [ 'setProperties', 'identify', 'track' ];
        var roundtripUrl = "https://s.adroll.com/j/" + adroll_adv_id
                + "/roundtrip.js";
        for (a = 0; a < w.adroll.f.length; a++) {
            w.adroll[w.adroll.f[a]] = w.adroll[w.adroll.f[a]] || (function(n) {
                return function() {
                    w.adroll.push([ n, arguments ])
                }
            })(w.adroll.f[a])
        }

        e = d.createElement('script');
        o = d.getElementsByTagName('script')[0];
        e.async = 1;
        e.src = roundtripUrl;
        o.parentNode.insertBefore(e, o);
    })(window, document);
    adroll.track("pageView");
</script>

**Pull request merged**